circular_linked_list/circular_linked_list.cpp

```diff
@@ -0,0 +1,44 @@
+#include<iostream>
+using namespace std;
+struct node{
+    int data;
+    node* next;
+};
+
+void show( node *head ) {
+    node *current = head;
+    cout<<"The list is ";
+    do {
+        cout << current->data << " ";
+        current = current->next;
+    } while(current != head);
```

Comment:
Extra space after ( in function call  [whitespace/parens] [4]

*Origin: CPPLintBear, Section: `all.cpp`.*